Source/FullScreenPokemon.ts

```diff
@@ -6667,6 +6667,76 @@ module FullScreenPokemon {
                     }));
         }
 
+        /**
+         * Cutscene for a Scratch attack in battle.
+         * 
+         * @param FSP
+         * @param settings   Settings used for the cutscene.
+         * @param args   Settings for the routine.
+         */
+        cutsceneBattleAttackScratch(FSP: FullScreenPokemon, settings: IBattleCutsceneSettings, args: IBattleAttackRoutineSettings): void {
+            let defenderName: string = args.defenderName;
+            let defender: IThing = <IThing>FSP.BattleMover.getThing(defenderName);
+            let dt: number = 6;
+            let direction: number = defenderName === "opponent" ? -1 : 1;
+            let scratchLine: string = direction === -1 ? "ScratchLine" : "ScratchLineInverted";
+            let startX: number;
+            let startY: number;
+            let differenceX: number = 43;
+            let lineArray: IThing[] = [];
+
+            if (direction === -1) {
+                startX = 595;
+                startY = 231;
+            } else {
+                startX = 422;
+                startY = 318;
+            }
```

Comment:
These should be fixed positions relative to where the enemy area is. See how other battle attack animations choose their `startX` equivalents.